geopandas/io/arrow.py

```diff
@@ -363,6 +363,24 @@ def _get_filesystem_path(path, filesystem=None, storage_options=None):
     return filesystem, path
 
 
+def _ensure_arrow_fs(filesystem):
+    from pyarrow import fs
+
+    if isinstance(filesystem, fs.FileSystem):
+        return filesystem
+
+    # handle fsspec-compatible filesystems
+    try:
+        import fsspec
+    except ImportError:
+        pass
```

Comment:
This should pass if fsspec isn't installed. This was partly copy pasted from a helper function in pyarrow: https://github.com/apache/arrow/blob/01e4ad095a7649afc7a7316447bc5cc1fd9679a2/python/pyarrow/fs.py#L103-L113. That helper function is more generic, so it might be it could be simplified here. But in general we still want to be able to check whether `filesystem` is an fsspec object, without requiring fsspec to be availble to use this function.